code/addons/interactions/template/stories/basics.stories.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable jest/no-standalone-expect */
+import globalThis from 'global';
+import { within, waitFor, fireEvent, userEvent } from '@storybook/testing-library';
+import { expect } from '@storybook/jest';
+
+export default {
+  component: globalThis.Components.Form,
```

Comment:
Why aren't you importing `Form`?

Reply:
@ghengeveld this is our mechanism to allow the same "generic" stories to be used in different renderers/frameworks. By using a component attached to global it can be implemented in each renderer separately. In this PR I've only implemented the react version so far.